.github/workflows/add_to_octokit_project.yml

```diff
@@ -6,6 +6,8 @@ on:
   pull_request_target:
     types: [reopened, opened]
 
+permissions: {}
```

Comment:
Hey @JamieMagee this particular workflow is generated from the octokit org repo [here](https://github.com/octokit/.github/blob/main/.github/templates/add_to_octokit_project.yml).

Please make the changes there and then submit a PR. Once it's been merged, the workflow will publish the changes to all octokit / Terraform repos.

Reply:
Thanks. Will do. I'll revert the changes in this file until then.